web/modules/custom/dpl_library_agency/src/GeneralSettings.php

```diff
@@ -23,6 +23,7 @@ class GeneralSettings extends DplReactConfigBase {
   const EREOLEN_MY_PAGE_URL = 'https://ereolen.dk/user/me';
   const EREOLEN_HOMEPAGE_URL = 'https://ereolen.dk';
   const FBI_PROFILE = 'next';
+  const OPENING_HOURS_URL = '';
```

Comment:
Consider making `/branches` the default value.

#809 will create a list of branches which in turn will link to the individual branch with the corresponding opening hours. I think that would be a sensible default value.